content/_data/changelogs/lts.yml

```diff
@@ -1813,6 +1813,11 @@
   # issue: 51603 too minor
 - version: "2.138.1"
   date: 2018-09-12
+  banner: >
```

Comment:
No longer running on decade+ old, unsupported OSes does not warrant a banner.

Reply:
@daniel-beck RHEL 5 is on extended support almost till 2021: https://en.wikipedia.org/wiki/Red_Hat_Enterprise_Linux#Version_history_and_timeline

I can confirm that I know some companies using RHEL 5 in 2019 (in addition to the ticket)

Reply:
Great. They are why we're adding it to the upgrade guide. But I doubt this is common enough to warrant a banner.

Reply:
Yep, we have centos/rhel5 agents to build certain C++ apps and ran into this